consensus/consensus_v2.go

```diff
@@ -378,14 +384,14 @@ func (consensus *Consensus) onPrepare(msg *msg_pb.Message) {
 		Str("validatorPubKey", validatorPubKey.SerializeToHexStr()).Logger()
 
 	// proceed only when the message is not received before
-	signed := consensus.Decider.ReadSignature(quorum.Prepare, validatorPubKey)
+	signed := consensus.Decider.ReadSignature(quorum.Announce, validatorPubKey)
```

Comment:
why Announce here, instead of Prepare?

Reply:
making codebase consistent in naming because of how consensus code actually calls, but talking w/RJ some more, will change back name to prepare.